lib/tests/facts/schema.cc

```diff
@@ -560,6 +561,13 @@ void validate_attributes(YAML::Node const& node)
         REQUIRE_FALSE(caveats.empty());
     }
 
+    auto blockgroup_attribute = node["blockgroup"];
```

Comment:
Ideally I would also like to be able to test that each blockgroup attribute corresponds to the name of a resolver that has is_blockable() == true. But like elsewhere in this code base, these tests do not currently assume or validate any mapping between facts and specific resolvers. Since this attribute depends on that mapping, is that something we would want to start validating?